atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -100,11 +101,25 @@ def get_fail_reason(self):
             return {'general': 'build not started'}
         build_annotations = self.build.get_annotations() or {}
         metadata = json.loads(build_annotations.get('plugins-metadata', '{}'))
-        fail_reason = metadata.get('errors', {})
-
+        fail_reason = {}
         if self.monitor_exception:
             fail_reason['general'] = repr(self.monitor_exception)
 
+        try:
+            fail_reason.update(metadata['errors'])
+        except KeyError:
+            try:
+                build_name = self.build.get_build_name()
+                pod = self.osbs.get_pod_for_build(build_name)
+            except OsbsException:
+                pass
+            else:
+                try:
+                    fail_reason['pod'] = pod.get_failure_reason()
+                except AttributeError:
```

Comment:
Can we combine these try/except blocks somehow. Something about having 3 nested of these seems odd. Maybe:
```python
try:
  build_name = self.build.get_build_name()
  pod = self.osbs.get_pod_for_build(build_name)
  fail_reason['pod'] = pod.get_failure_reason()
except (OsbsException, AttributeError):
  pass
```